main.go

```diff
@@ -163,12 +165,19 @@ func getUnrestrictedClient() client.Client {
 	return k8sClient
 }
 
+// This function is a placeholder for offering plugin integration
+func pluginAddToScheme(scheme *runtime.Scheme) {
+	controllers.DFAddToScheme(scheme)
+	controllers.DFCAddToScheme(scheme)
+}
+
 func mapCRDAvailability(crdNames ...string) map[string]bool {
```

Comment:
As discussed over the last review, no need to pass a CRD name list
```suggestion
func mapCRDAvailability() map[string]bool {
```